docs/core/tutorials/cli-create-console-app.md

```diff
@@ -177,7 +177,7 @@ Hello World!
 As mentioned at the start of this article, an operating system-specific executable was created along with the `Hello.dll`. On Windows, this would be `Hello.exe`; on Linux or macOS, this would be `hello`. With the example above, the file is named with `Hello.exe` or `Hello`. You can run this published executable directly.
 
 ```console
-.\bin\Debug\netcoreapp3.1\Hello.exe
+bin\Debug\netcoreapp3.1\publish\Hello.exe
```

Comment:
There is another occurrence. And I think the ".\" is okay to be kept. I even think it's **necessary** so this will be compatible with both cmd and powershell. Without it, it will be compatible with cmd only.

Reply:
Good point. The other reference is before the publish action, so the path on that wouldn't change.

```suggestion
.\bin\Debug\netcoreapp3.1\publish\Hello.exe
```